org.kartkrew.RingRacers.yaml

```diff
@@ -0,0 +1,57 @@
+app-id: org.kartkrew.RingRacers
+runtime: org.freedesktop.Platform
+runtime-version: '23.08'
+sdk: org.freedesktop.Sdk
+command: ringracers.sh
+finish-args:
+  - --share=ipc
+  - --share=network
+  - --socket=fallback-x11
+  - --socket=pulseaudio
+  - --socket=wayland
+  - --device=all
+  - --persist=.ringracers
+  - --filesystem=xdg-run/app/com.discordapp.Discord:create
+  - --filesystem=xdg-run/discord-ipc-0
+
+modules:
+  - name: ringracers
+    buildsystem: cmake-ninja
+    builddir: true
+    sources:
+      - type: archive
+        url: https://github.com/KartKrewDev/RingRacers/archive/refs/tags/v2.1.tar.gz
+        sha256: 6980c398ca6ddb1e94cf60234483767390b396d693923437c98df8bd7d13b266
+      - type: file
```

Comment:
this should by `type: archive` so that you don't need to unzip.

Reply:
It's using type: file because type: archive doesn't have an option to control where it extracts to

Reply:
it does. it very well documented.

`strip-components`, `dest`, among others.